src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -749,6 +754,12 @@
         - name: --apiserver-subnet-id
           type: string
           short-summary: The ID of a subnet in an existing VNet into which to assign control plane apiserver pods(requires --enable-apiserver-vnet-integration)
+        - name: --enable-custom-ca-trust
+          type: bool
+          short-summary: Enable Custom CA Trust on agent node pool.
+        - name: --disable-custom-ca-trust
```

Comment:
Since you have an additional short form option named `--dcat` declared. This should be `- name: --disable-custom-ca-trust --dcat`.